en/starter/generator/project_layout_reference.md

```diff
@@ -26,20 +26,20 @@ The files that populate the application directory depend on the [project type](c
 The following structure is common to all generated applications:
 
 ```
-├── LICENSE                         - MIT license for the generated code
-├── Sources/                        - Swift source code for the project
-│   ├── Application/                - Swift module for application logic
-│   │   └── Application.swift       - Swift code for application logic
-│   └── <application_name>/         - Swift module for executable
-│       └── main.swift              - Swift code for executable
-├── Tests/                          - Swift source code for the tests
-│   ├── ApplicationTests/           - Swift test module
-│   │   └── RouteTests.swift        - Swift code for the test implementation
-│   └── LinuxMain.swift             - Swift test definition for Linux
-├── Package.swift                   - Swift Package Manager configuration file
-├── Package.resolved                - Swift Package Manager resolved packages
-├── config.json                     - Application configuration file
-├── spec.json                       - Generator specification file for the project
+├── LICENSE                           - MIT license for the generated code
+├── Sources/                          - Swift source code for the project
+│   ├── Application/                  - Swift module for application logic
+│   │   └── Application.swift         - Swift code for application logic
+│   │   └── InitialisationError.swift - Swift code for initialisation error
```

Comment:
The actual file which is generated is called `InitializationError.swift` so the s needs to change to a z in wherever it is referenced in the documentation.